datamodels/2.x/itop-attachments/dictionaries/nl.dict.itop-attachments.php

```diff
@@ -40,12 +40,12 @@
 	'Attachments:NoAttachment' => 'Geen bijlage. ',
 	'Attachments:PreviewNotAvailable' => 'Er is geen voorbeeld beschikbaar voor dit type bijlage.',
 	'Attachments:Error:FileTooLarge' => 'Het bestand is te groot om geüpload te worden: %1$s',
-	'Attachments:Error:UploadedFileEmpty' => 'The received file is empty and cannot be attached.
-Either you have pushed an empty file,
-or ask your iTop administrator if the iTop server disk is full.~~',
+	'Attachments:Error:UploadedFileEmpty' => 'Het bestand is leeg en kan niet worden toegevoegd.
+Mogelijk heb je een leeg bestand geüpload,
+of vraag de iTop administrator om na te kijken of de schijf van de iTop server vol zit',
```

Comment:
```suggestion
of vraag de iTop administrator om de opslag van de iTop server na te kijken',
```
This eliminates the double use of "of".